app/views/users/index.html.erb

```diff
@@ -2,4 +2,10 @@
   People of MRV
 <% end %>
 
+<% if current_user.admin? or current_user.facilitator? %>
+	<% content_for :page_header_action do %>
+		<%=link_to 'Export Users to CSV', users_path(request.parameters.merge({format: 'csv'})), class: "button is-primary"%>
```

Comment:
My only note on this one is that if you export the CSV and then refresh the page, you will re-download the CSV. It doesn't happen on the cohorts page because the route is `cohorts/4.csv` rather than a query string with search parameters. I don't have a clear solution for this in mind yet, but will think about it!